connector/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/MySQLIntegrationSuite.scala

```diff
@@ -246,13 +257,6 @@ class MySQLIntegrationSuite extends DockerJDBCIntegrationSuite {
     checkAnswer(df, Row(true, true, true))
     df.write.mode("append").jdbc(jdbcUrl, "bools", new Properties)
     checkAnswer(df, Seq(Row(true, true, true), Row(true, true, true)))
-    val mb = new MetadataBuilder()
```

Comment:
Removed this because the schema checking is not very necessary, and Maria/MySQL Connector/Js behave different